.github/workflows/CI-python.yml

```diff
@@ -11,7 +11,13 @@ jobs:
     strategy:
       matrix:
         packageDirectory:
-          ["rai_core_flask", "responsibleai", "erroranalysis", "raiutils"]
+          [
+            "rai_core_flask",
+            "responsibleai",
+            "erroranalysis",
+            "raiutils",
+            "rai_test_utils"
```

Comment:
We seem to have some inconsistent naming in packages - raiutils has no underscore, rai_core_flask has underscores, erroranalysis, raiwidgets and responsibleai have no underscores.  I'm wondering if we can find some general philosophy or understanding as to why, and how our packages should be named moving forward, especially since raiutils and rai_test_utils are so similar but the names are quite different.

Reply:
I tried to look at the readability of the package/names or namespaces "raitestutils" and "rai_test_utils". I think the latter read more easily than the first one. raiutils and rai_test_utils are quite different in their scope, in my opinion.